go/src/koding/klient/machine/index/change.go

```diff
@@ -180,6 +199,12 @@ func (c *Change) Coalesce(newer *Change) *Change {
 	}
 }
 
+// String implements fmt.Stringer interface. It pretty prints stored change.
+func (c *Change) String() string {
+	age := time.Now().UTC().Sub(time.Unix(0, atomic.LoadInt64(&c.createdAt)))
```

Comment:
just curious why do you need to be atomic here? also could you use `c.CreatedAtUnixNano()`